src/modules/UI/scenes/Plugins/api.js

```diff
@@ -4,18 +4,21 @@ import type { EdgeReceiveAddress, EdgeTransaction } from 'edge-core-js'
 import { Linking } from 'react-native'
 import { Actions } from 'react-native-router-flux'
 
+import ENV from '../../../../../env.json'
 import { SEND_CONFIRMATION } from '../../../../constants/SceneKeys.js'
 import { type GuiMakeSpendInfo } from '../../../../reducers/scenes/SendConfirmationReducer.js'
 import * as WALLET_API from '../../../Core/Wallets/api'
 
-const formatWallet = w => ({
-  id: w.id,
-  name: w.name,
-  type: w.type,
-  currencyCode: w.currencyCode,
-  primaryNativeBalance: w.currencyCode,
-  fiatCurrencyCode: w.fiatCurrencyCode
-})
+const formatWallet = w => {
+  return {
+    id: w.id,
+    name: w.name,
+    type: w.type,
+    currencyCode: w.currencyCode ? w.currencyCode : w.currencyInfo.currencyCode,
+    primaryNativeBalance: w.currencyCode,
+    fiatCurrencyCode: w.fiatCurrencyCode
```

Comment:
Just want to point out that `primaryNativeBalance` should not have anything to do with a `currencyCode`

Reply:
fixed, was copy past error from previous file